### ISSUE TITLE
Add File coverage to format listings

### ISSUE THREAD

**Issue by nkrabben** [digipres/sentinel#11]
The file tool also identifies a number of formats and would be useful to have in this combined registry.

**Comment by anjackson** [comment]
Yes, very happy to review any pull requests to improve this. You can see the _fine free file_ magic here: https://github.com/file/file/tree/master/magic/Magdir

It doesn't have file extensions, but it does have some MIME types, e.g. https://github.com/file/file/blob/master/magic/Magdir/pdf

**Comment by anjackson** [comment]
Hmm, does have some extension info, e.g.

https://github.com/file/file/blob/d3516f29966c52ebd0619839110edd6646540e0b/magic/Magdir/jpeg#L17-L21

But it looks a bit gnarly to parse. I guess look for the `!:mime` and then backtrack to a non-`!:` line and backtrack via the `>` to put the name together?

**Comment by anjackson** [comment]
See https://github.com/digipres/digipres.github.io/pull/60 - I've managed to make a reasonably useful export of the file magic test tree, using https://github.com/trailofbits/polyfile 

It's unfortunately a one-off, as `polyfile` is a couple of years behind being fully compatible with `file`, so I can't use it to track the latest changes. But still, it's a useful addition for analysis.

**Comment by anjackson** [comment]
Note that `polyfile` has been updated and should now work with more recent releases: trailofbits/polyfile#3374

Older version is hooked in via 

https://github.com/digipres/sentinel/blob/de2a1efb1e77f4acb2aa1d82c872de1f94b0e786/pyproject.toml#L12

and means this could be uncommented

https://github.com/digipres/digipres.github.io/blob/00b9aea89172fde594c6e0c0d654f2204a286162/_sources/registries/file/download-polyfile.py#L12

which would allow the index to be updated from the file git tree rather than the one built into `polyfile`.
